packages/terra-table/src/TableSubheader.jsx

```diff
@@ -19,21 +22,14 @@ const TableSubheader = ({
   content,
   colSpan,
   ...customProps
-}) => {
-  const contentClassName = classNames([
-    'terra-Table-subheader',
-    customProps.className,
-  ]);
-
+}) => (
   // count is based on the number of columns and assigned in the table component which contains this subheader
-  return (
-    <tr className="terra-Table-subheaderRow">
-      <td {...customProps} className={contentClassName} colSpan={colSpan}>
-        {content}
-      </td>
-    </tr>
-  );
-};
+  <tr data-class="subheader-row">
+    <td {...customProps} className={cx('subheader', customProps.className)} colSpan={colSpan}>
```

Comment:
customProps.className have been placed within the cx method for serveral components. If the custom class name is not locally scoped, it will not be name spaced/compiled with the css modules naming. 

I added a custom class name to a table nightwatch example and the result was this:

![screen shot 2017-07-26 at 8 31 40 am](https://user-images.githubusercontent.com/14099737/28624060-7c58c1fe-71dd-11e7-879f-0c07fd9de1f1.png)